fractal-puppy/Engine
# Experience improvement

- Now every enemy leaves a sphere after dying that will go towards the player. When the player touches the sphere, he gets the experience.
- A dissolve effect has been added to the player, so when he gets experience it is played.

[Example](https://gyazo.com/ed616620ccc6638953b1fcedd7ad2a63)

**To test:**
- Run the engine and play the first level.
- Kill an enemy and see if the sphere appears and does something similar to what is shown in the example.
